sdk/eventhub/event-hubs/changelog.md

```diff
@@ -1,3 +1,28 @@
+### 2019-08-06 5.0.0-preview.2
+
+#### General
+- The current `Sender` does not have a provision of "awaiting" on sending a message. We need to add handlers on the `Sender` for accepted, rejected, released, modified to ensure whether the message was successfully sent. 
+  Now, we use a new `AwaitableSender` instead of `Sender` which adds the handlers internally and provides an awaitable `send()` operation.
+- Refactor receiver that allows the same amqp link to be shared between streaming and batching receivers.
+- All time related entites have been updated to use milli seconds as the unit of time for consistency.
+- New error InsufficientCreditError is introduced for the scenario where rhea is unable to send events due to its internal buffer being full. This is a transient error and so is treated as retryable.
+- The error OperationTimeoutError was previously mistakenly classified as an AMQP error which is now corrected. Since this can also be a transient error, it is treated as retryable.
+- Update the version of rhea-promise to "^1.0.0".
+- Update the version of ws to "^7.1.1".
+
+#### Publishing events
+- Introduced the `EventDataBatch`, allowing for publication of a batch of events with known size constraint. 
+  This is intended to ensure that publishers can build batches without the potential for an error when sending and to allow publishers with bandwidth concerns to control the size of each batch published.
```

Comment:
```suggestion
  This allows publishers to build batches without the potential for the `MaxSizeExceededError` error when sending events. It also allows publishers with bandwidth concerns to control the size of each batch published.
```

confirm the name of the error above with @chradek

Reply:
Should be MessageTooLargeError: https://github.com/Azure/azure-sdk-for-js/blob/0c17ebe2751f1fd8f480f76e2bd8fa8056c2c61c/sdk/eventhub/event-hubs/test/sender.spec.ts#L439